index.js

```diff
@@ -13,6 +13,7 @@ const MAX_TAG_VALUE_LENGTH = 256;
 const SANITIZATION_CHARACTER = '_';
 const ROLE_SESSION_NAME = 'GitHubActions';
 const REGION_REGEX = /^[a-z0-9-]+$/g;
+const AWS_CURRENT_REGION = process.env.AWS_REGION
```

Comment:
or should this be `AWS_DEFAULT_REGION` ?

Reply:
I don't think so; you want to be able to override the default region, which is what `AWS_REGION` is for